front/src/applications/stdcm/components/StdcmForm/StdcmConsist.tsx

```diff
@@ -102,6 +107,13 @@ const StdcmConsist = ({ isDebugMode, disabled = false }: StdcmConsistProps) => {
   const lengthFieldStatus = createFieldStatus('totalLength');
   const speedFieldStatus = createFieldStatus('maxSpeed');
 
+  const getMissingFieldMessage = (value?: number): string | undefined => {
+    if (value === undefined || value === null) {
```

Comment:
Value can't be null. 0 is not accepted in the inputs (always replaced by undefined), so we should be good with this
```suggestion
    if (!value) {
```

Reply:
done